s3transfer/upload.py

```diff
@@ -521,7 +521,12 @@ class UploadSubmissionTask(SubmissionTask):
         'ExpectedBucketOwner',
     ]
 
-    COMPLETE_MULTIPART_ARGS = ['RequestPayer', 'ExpectedBucketOwner']
+    COMPLETE_MULTIPART_ARGS = [
+        'RequestPayer',
+        'ExpectedBucketOwner',
+        'SSECustomerKey',
+        'SSECustomerAlgorithm',
```

Comment:
Do we need to proxy over the `SSECustomerKeyMD5` to the complete MPU as well? I noticed that it was in the upload parts list but not the complete mpu list.

Reply:
There was back and forth on this. They said only the Key/Algo for now. I've added what we know we need and we can always extend if there is further feedback. S3 has some knowledge gaps on the full feature scope, so we're meeting the known use case.

Reply:
Added it for good measure. After some testing it doesn't have any impact on the call so we might as well be complete and pass it if it was provided.